addon/services/file-manager.js

```diff
@@ -141,25 +244,65 @@ export default Ember.Service.extend({
                 headers: {
                     Authorization: `Bearer ${accessToken}`
```

Comment:
It seems that this would bake the authorization mechanism into a service.

In the near future, we may want to make authorization type configurable, so that we (COS) can talk to the OSF via cookies. Will waterbutler always be using a token, or will the service need to support cookie auth too?